examples/mipmap/main.rs

```diff
@@ -134,7 +134,7 @@ impl Example {
             address_mode_u: wgpu::AddressMode::ClampToEdge,
             address_mode_v: wgpu::AddressMode::ClampToEdge,
             address_mode_w: wgpu::AddressMode::ClampToEdge,
-            mag_filter: wgpu::FilterMode::Nearest,
+            mag_filter: wgpu::FilterMode::Linear,
             min_filter: wgpu::FilterMode::Linear,
```

Comment:
let's switch `min_filter` to `Nearest` to follow the spirit of the original code